src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,20 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    {user
+      && (
+        <a
+          key={user.id}
+          className="UserInfo"
+          href={`mailto:${user.email}`}
+        >
+          {user.name}
+        </a>
+      )}
+  </>
```

Comment:
You check user in TodoInfo
After that check, he will always be here
And destructure `user`
```suggestion
  <a
          key={user.id}
          className="UserInfo"
          href={`mailto:${user.email}`}
        >
          {user.name}
        </a>
```